portal/views/portal.py

```diff
@@ -429,15 +430,22 @@ def admin():
     """user admin view function"""
     # can't do list comprehension in template - prepopulate a 'rolelist'
 
-    request_org_list = request.args.get('org_list', None)
+    user = current_user()
+
+    pref_org_list = None
+    # check user table preference for organization filters
+    pref = TablePreference.query.filter_by(table_name='adminList',
+                                           user_id=user.id).first()
+    if pref and pref.filters:
+        pref_org_list = pref.filters.get('orgs_filter_control')
```

Comment:
You can remove the below condition (`if pref_org_list`) and un-indent the remaining code if you set the default `pref_org_list` to an empty string by default.
eg:
```
pref_org_list = pref.filters.get('orgs_filter_control', '')
org_list = set()
....
```

Reply:
@ivan-c, ivan, didn't fix here as an else clause is attached to this statement to perform default action if the if clause fails here (in this case pref_org_list is null or empty)